tests/storage_test.go

```diff
@@ -248,7 +250,83 @@ var _ = Describe("Storage", func() {
 			})
 
 		})
+		Context("Run a VMI wiht VirtIO-FS and a datavolume", func() {
+			var dataVolume *cdiv1.DataVolume
+			BeforeEach(func() {
+				if !tests.HasCDI() {
+					Skip("Skip DataVolume tests when CDI is not present")
+				}
+				dataVolume = tests.NewRandomDataVolumeWithHttpImport(tests.GetUrl(tests.AlpineHttpUrl), tests.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				tests.EnableFeatureGate(virtconfig.VirtIOFSGate)
+			})
+
+			AfterEach(func() {
+				err := virtClient.CdiClient().CdiV1alpha1().DataVolumes(dataVolume.Namespace).Delete(dataVolume.Name, &metav1.DeleteOptions{})
+				Expect(err).To(BeNil())
+				tests.DisableFeatureGate(virtconfig.VirtIOFSGate)
+			})
+			It("should be successfully started and viriofs could be accessed", func() {
+				tests.SkipPVCTestIfRunnigOnKindInfra()
+				waitForDataVolume := func(dataVolume *cdiv1.DataVolume) {
+					Eventually(func() cdiv1.DataVolumePhase {
+						dv, err := virtClient.CdiClient().CdiV1alpha1().DataVolumes(dataVolume.Namespace).Get(dataVolume.Name, metav1.GetOptions{})
+						Expect(err).ToNot(HaveOccurred())
+
+						return dv.Status.Phase
+					}, 60*time.Second, 1*time.Second).
+						Should(Equal(cdiv1.ImportInProgress), "Timed out waiting for DataVolume to enter ImportInProgress phase")
```

Comment:
this line ended up being flaky.

The import happens so quickly, it's possible for the import to move from InProgress to Succeeded before the next polling interval occurs.

I think the easiest way to fix this is to wait for Succeeded insteam of ImportInProgress.

Reply:
Done, thanks!